apps/axistream/mold_example/mold_example.cc

```diff
@@ -0,0 +1,44 @@
+#include <cstdint>
+#include "mold_example.hh"
+#include "axi_stream_reader.hh"
+#include "ethernet.hh"
+#include "ip.hh"
+#include "udp.hh"
+#include "moldudp64.hh"
+#include "itch.hh"
+
+void mold_example(AxiStream<8>& in_stream, hls::stream<int32_t>& out) {
```

Comment:
@stephenneuendorffer I made the mold_remover_stream example (https://github.com/Xilinx/HLS_packet_processing/blob/efbd9996fbc5568925e2d851c64f5d2106657d51/apps/mold_remover_stream/app.cpp#L36-L125)